workflows/downloadraw-workflow.yaml

```diff
@@ -18,58 +23,81 @@ spec:
     operator: "Equal"
     value: "spot"
     effect: "NoSchedule"
+  parallelism: 50  # Limit number of parallel jobs.
```

Comment:
What's the rationale behind picking this number? Not questioning your decision, just want to better understand the reasoning behind it.

Reply:
@dgergel, no specific reason. I don't want to blast their server with download requests or flood any of our nodes with IO. Wanted to pick a nice and easy number so this downloads at a steady rate as we add more GCMs to this list. 50 feels like a good round number to start.